db/migrate/20181017200423_add_service_offering_ref_to_portfolio_item.rb

```diff
@@ -0,0 +1,8 @@
+class AddServiceOfferingRefToPortfolioItem < ActiveRecord::Migration[5.1]
+  def change
+    say_with_time "Removing all PortfolioItems, adding a new required field: service_offerings_ref" do
+      PortfolioItem.delete_all
+    end
+    add_column :portfolio_items, :service_offering_ref, :string, :null => false
```

Comment:
We already have the `null => false` constraint defined at the model level, which will allow for some flexibility if we need it in the future, I would avoid adding it here as well.

Seems similar to `default_value` issues we have discussed on other MIQ repos.  We prefer to define those at the model level for the same reason.